pkgs/development/python-modules/sslyze/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, callPackage, fetchPypi, fetchFromGitHub, buildPythonPackage, nassl, cryptography, cryptography_vectors, typing-extensions, faker }:
+buildPythonPackage rec {
+  name = "sslyze";
+  version = "3.0.4";
+  src = fetchFromGitHub {
+    owner = "nabla-c0d3";
+    repo = name;
+    rev = version;
+    sha256 = "19fnnbdmpfdpmadqqcnj79hh72wrjiahn58rbxis4586ydyi552g";
+  };
+
+  patchPhase = ''
+    sed -i 's/cryptography>=2.6,<=2.9/cryptography>=2.6,<=${cryptography.version}/g' setup.py 
```

use nix substitute helper, and semver version bounds
```suggestion
    substituteInPlace setup.py \
      --replace "cryptography>=2.6,<=2.9" "cryptography>=2.6,<=3"
```